src/utils/logger.ts

```diff
@@ -1,23 +1,37 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-import { pino, Logger } from 'pino'
-import { LogLevel, BaseLogger } from '@credo-ts/core'
+import { pino, Logger, LevelWithSilent } from 'pino'
+import { LogLevel as CredoLogLevel, BaseLogger } from '@credo-ts/core'
+
+const tsLogLevelMap = {
+  silent: CredoLogLevel.off,
+  trace: CredoLogLevel.trace,
+  debug: CredoLogLevel.debug,
+  info: CredoLogLevel.info,
+  warn: CredoLogLevel.warn,
+  error: CredoLogLevel.error,
+  fatal: CredoLogLevel.fatal,
+} as const
+
+const invTsLogLevelMap = {
+  [CredoLogLevel.off]: 'silent' as const,
+  [CredoLogLevel.test]: 'trace' as const,
+  [CredoLogLevel.trace]: 'trace' as const,
+  [CredoLogLevel.debug]: 'debug' as const,
+  [CredoLogLevel.info]: 'info' as const,
+  [CredoLogLevel.warn]: 'warn' as const,
+  [CredoLogLevel.error]: 'error' as const,
+  [CredoLogLevel.fatal]: 'fatal' as const,
+} as const
+
+export type LogLevel = LevelWithSilent
 
 export default class PinoLogger extends BaseLogger {
   private logger: Logger
 
   // Map our log levels to tslog levels
-  private tsLogLevelMap = {
-    [LogLevel.test]: 'silent', // pino does not have 'silly' so used silent for .test() method
-    [LogLevel.trace]: 'trace',
-    [LogLevel.debug]: 'debug',
-    [LogLevel.info]: 'info',
-    [LogLevel.warn]: 'warn',
-    [LogLevel.error]: 'error',
-    [LogLevel.fatal]: 'fatal',
-  } as const
 
-  public constructor(logLevel: LogLevel) {
-    super(logLevel)
+  public constructor(logLevel: LevelWithSilent) {
+    super(tsLogLevelMap[logLevel])
 
     this.logger = pino(
       {
```

Comment:
i may be getting confused but wasn't the original ticket to make sure it is loaded from the env here on line 40?

Reply:
🤦‍♂️

Reply:
fixed